osu.Game/Overlays/Settings/SettingsItem.cs

```diff
@@ -70,27 +70,32 @@ public virtual LocalisableString LabelText
         }
 
         /// <summary>
-        /// Text to be displayed at the bottom of this <see cref="SettingsItem{T}"/>.
-        /// Generally used to recommend the user change their setting as the current one is considered sub-optimal.
+        /// Clear any warning text.
         /// </summary>
-        public LocalisableString? WarningText
+        public void ClearWarningText()
         {
-            set
-            {
-                bool hasValue = value != default;
-
-                if (warningText == null)
-                {
-                    if (!hasValue)
-                        return;
+            warningText?.Expire();
+            warningText = null;
+        }
 
-                    // construct lazily for cases where the label is not needed (may be provided by the Control).
-                    FlowContent.Add(warningText = new SettingsNoticeText(colours) { Margin = new MarginPadding { Bottom = 5 } });
-                }
+        /// <summary>
+        /// Set the text to be displayed at the bottom of this <see cref="SettingsItem{T}"/>.
+        /// Generally used to recommend the user change their setting as the current one is considered sub-optimal.
+        /// </summary>
+        /// <param name="text">The text to display.</param>
+        /// <param name="isWarning">Whether the text is in a warning state. Will decide how this is visually represented.</param>
+        public void SetWarningText(LocalisableString text, bool isWarning = true)
```

Comment:
Having `warning` in the method name but allow setting a green note instead feels counter-intuitive, would rename it to something like `SetNote`/`SetNoteText` and default `isWarning` to false, or remove the default.

Reply:
Have applied this change.